client_chargeauthtokens.go

```diff
@@ -22,6 +51,33 @@ type ListChargeAuthTokensResponse struct {
 	PageMeta PageMeta `json:"meta"`
 }
 
+// CreateChargeAuthToken to create a new charge auth token.
+func (c *clientImpl) CreateChargeAuthToken(
+	ctx context.Context,
+	request CreateChargeAuthTokenRequest,
+) (*ChargeAuthToken, error) {
+	path := "/v1/charge-auth-tokens"
```

Comment:
Should we have this base path as a constant so that we don't have to declare it in different methods?